src/components/InlineSvg/InlineSvg.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { Icon, IconName } from '../Icon';
+import { Skeleton, SkeletonVariant } from '../Skeleton';
+
+import { InlineSvgProps } from './InlineSvg.types';
+
+export const InlineSvg = ({
```

Comment:
Please update to function component with `forwardRef`, so other components might use `ref.current `to access the `<div>` to aid more fiddly implementations. This will also present the opportunity to support `...rest` attributes of the `div` element.